app/src/main/java/de/danoeh/antennapod/adapter/QueueRecyclerAdapter.java

```diff
@@ -209,9 +211,26 @@ public void bind(FeedItem item) {
             FeedMedia media = item.getMedia();
 
             title.setText(item.getTitle());
+            int format = DateUtils.FORMAT_ABBREV_ALL;
+            if(item.publishedWithinLastYear()) {
+                format |= DateUtils.FORMAT_NO_YEAR;
+            }
             String pubDateStr = DateUtils.formatDateTime(mainActivity.get(),
-                item.getPubDate().getTime(), DateUtils.FORMAT_ABBREV_ALL);
-            pubDate.setText(pubDateStr.replace(" ", "\n"));
+                item.getPubDate().getTime(), format);
+            int index = 0;
+            if(StringUtils.countMatches(pubDateStr, ' ') == 1 || StringUtils.countMatches(pubDateStr, ' ') == 2) {
```

Comment:
Added this after changing the language to Chinese. It showed something like "1 [Chinese symbol] 15 [Chinese symbol]". Decided I don't want to mess with it and desist from any wrapping.